### PR TITLE
Update Variation and VEP public documentation for e106

### PR DESCRIPTION
**Pull request by nakib103**

### CODE REVIEW

File: docs/htdocs/info/docs/tools/vep/script/vep_cache.html [hunk 1]
```diff
@@ -169,12 +169,12 @@ <h3 id="cache_content">Data in the cache</h3>
...
       <tr>
         <td><b>GENCODE</b></td>
-        <td>39</td>
+        <td>40</td>
         <td>19</td>
       </tr>
       <tr>
         <td><b>RefSeq</b></td>
-        <td>2020-12-10<br />(GCF_000001405.39_GRCh38.p13_genomic.gff)</td>
+        <td>2021-5-14<br />(GCF_000001405.39_GRCh38.p13_genomic.gff)</td>
```
Comment (helensch) [inline docs/htdocs/info/docs/tools/vep/script/vep_cache.html:177]: Suggest using the format 2021-05-14

File: ensembl/htdocs/info/genome/variation/species/sets.html [hunk 2]
```diff
@@ -151,7 +151,7 @@ <h2 id="Homo_sapiens" style="display:inline;color:#333">Human<span class="small
   <tr>
     <td><ul style="margin:0px"><li style="margin:0px">1000 Genomes 3 - SAS - common</li></ul></td>
     <td>1kg_3_sas_com</td>
-    <td>Variants genotyped in South Asian individuals by the 1000 Genomes project (phase 3) with frequency of at least 1%</td>
+    <td>Variants genotyped in East Asian individuals by the 1000 Genomes project (phase 3) with frequency of at least 1%</td>
```
Comment (helensch) [inline ensembl/htdocs/info/genome/variation/species/sets.html:154]: Please check this change - SAS should be "South Asian"

File: ensembl/htdocs/info/genome/variation/species/sets.html [hunk 3]
```diff
@@ -115,7 +115,7 @@ <h2 id="Homo_sapiens" style="display:inline;color:#333">Human<span class="small
...
```
Comment (helensch) [inline ensembl/htdocs/info/genome/variation/species/sets.html:118]: Please check this change SAS should be "South Asian"

**Review by helensch — COMMENTED** [review]
In the VEP_script_documentation.pdf 
- the "What New" section the release date of e106 needs to be updated
- In the previous version history the information for e105 needs to be added

-

**Comment by nakib103** [comment]
Hello @helensch , thanks for review.
I have updated the files accordingly.

**Comment by helensch** [comment]
Hi @nakib103 thanks for the updates. Looks good to me. @sarahhunt will review tomorrow

**Comment by sarahhunt** [comment]
Thanks @nakib103 & @helensch

**Pull request merged**